index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lista zakupów</title>
+    <link
+        rel="stylesheet" 
+        href="./style.css" 
+    />
+</head>
+
+
+<body>
+    
+        <header class="site-header">
+            
+                <h1>Lista zakupów</h1>
+            
+        </header>
+    
+        
+        <main>
+            <div class="shopping-card">
+                
+                <article class="shopping-card-item">
+
+                    <header class="shopping-card__header1">
+                        <div class="transparent-box">
+                            <h1 class="shopping__primary-header">
+                                Owoce
+                            </h1>
+                        </div>
+                        
+        
+                        
+                    </header>
+        
+                    <main class="shopping__body">
```

Comment:
tylko jedno "main" jest dozwolone na stronie, to bezposrednio w body

tutaj powinien byc zwykly div